tests/data/shrink_cat.py

```diff
@@ -8,17 +8,24 @@
                     'base_PixelFlags_flag_cr', 'base_PixelFlags_flag_bad', 'base_PixelFlags_flag_edge',
                     'base_PsfFlux_snr', 'base_PsfFlux_mag', 'base_PsfFlux_magErr', 'base_GaussianFlux_mag',
                     'base_GaussianFlux_magErr', 'base_GaussianFlux_snr', 'e1', 'e2', 'psf_e1', 'psf_e2',
-                    'filt']
+                    'filt', 'detect_isPrimary']
 
+def shrinkCat(infile, outfile, verbose=False):
+    cat = SimpleCatalog.readFits(infile)
+    for name in cat.schema.getNames():
+        if name in required_columns or 'flag' in name:  # Flags should compress anyway
+            continue
+        if verbose:
+            print(f"Shrinking {name}")
+        try:
+            cat[name][:] = np.repeat(0, len(cat))
+        except ValueError as e:
+            print(e)
 
-cat = SimpleCatalog.readFits(sys.argv[1])
-for name in cat.schema.getNames():
-    if name in required_columns or 'flag' in name:  # Flags should compress anyway
-        continue
-    print(f"Doing {name}")
-    try:
-        cat[name][:] = np.repeat(0, len(cat))
-    except ValueError as e:
-        print(e)
+    cat.writeFits(outfile)
 
-cat.writeFits(sys.argv[2])
+
+if __name__ == "__main__":
+    infile = sys.argv[1]
+    outfile = sys.argv[2]
+    shrinkCat(infile, outfile)
```

Comment:
Is this a flake8 error I see? (Reminder to run `scons`.) :)

Reply:
Thanks. I did run `scons` but didn't see this among the output. I run `flake8` separately now and verify no more of flake8 violations.